docs/ChangeLog.md

```diff
@@ -1,7 +1,21 @@
 # Splunk Operator for Kubernetes Change Log
 
+## 2.2.0 (2023-01-17)
+
+* This is the 2.2.0 release. The Splunk Operator for Kubernetes is a supported platform for deploying Splunk Enterprise with the prerequisites and constraints laid out [here](https://github.com/splunk/splunk-operator/blob/master/docs/README.md#prerequisites-for-the-splunk-operator)
```

Comment:
Shouldn't the link be "https://github.com/splunk/splunk-operator/blob/main/docs/README.md#prerequisites-for-the-splunk-operator" ? I mean main instead of master?

Reply:
Will correct this. FYI I tried both, `master` redirects to `main`